FlashEnglish/Sources/Views/Score/ScoreView.swift

```diff
@@ -8,12 +8,17 @@
 import SwiftUI
 
 struct ScoreView: View {
+
+
+
+
+
     // MARK: - Properties
     @EnvironmentObject var quizManager: QuizManager
-    @EnvironmentObject var navigationManager: NavigationManager
+            @EnvironmentObject var navigationManager: NavigationManager
 
     // MARK: - Body
-    var body: some View {
+    var body: some View             {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Masaya1582/FlashEnglish/blob/f0c1bb17c92f6625d748b7be15707d1134141005/FlashEnglish/Sources/Views/Score/ScoreView.swift#L21"></span>Opening braces should be preceded by a single space and on the same line as the declaration<br />
<code>opening_brace</code> <code>ScoreView.swift:21</code></td>
    </tr>
  </tbody>
</table>